### DISCUSSION TITLE
Update in Windows

### DISCUSSION THREAD

**Discussion by kenfabioh** [nzbgetcom/nzbget#133] [General]
How can I append/update my standard nzbget with these modifications? How can I build this? I'm on Windows

**Comment by TylerHomeLab** [comment]
Same question.  I'm installed the same way.  I want to point to this project instead and turn on updates.

**Comment by mkanet** [comment]
I am also very interested.

**Comment by JPetovello** [comment]
Me as well

**Comment by mkanet** [comment]
Aren't there more NzbGet Windows users than any other OS?

**Reply by paul-chambers** [reply]
I don't know, but there are certainly fewer Windows developers willing to help maintain nzbget ;)

**Comment by luckedea** [comment]
Closing this discussion as not vital - we build all necessary binaries/installers and maintain them.
